pd-client/client_test.go

```diff
@@ -175,16 +186,65 @@ func (s *testClientSuite) TestGetRegion(c *C) {
 	c.Assert(leader, DeepEquals, peer)
 }
 
+func (s *testClientSuite) TestGetPrevRegion(c *C) {
```

Comment:
I think best do this test as a unit test about `searchPrevRegion`. It easy to fail in ci env.